src/protocol/primitives/uuid.spec.ts

```diff
@@ -27,6 +28,13 @@ describe('Uuid', () => {
     expect(writeBuffer.toBuffer()).toEqual(uuidBytes);
   });
 
+  it('should create Uuid from UUID', () => {
+    const uuid = randomUUID();
+    const expectedUuid = Buffer.from(uuid.replace(/-/g, ''), 'hex');
```

Comment:
I see no benefit from using random values instead of hard-coded examples and expected results as in other test scenarios. Moreover, this causes a duplication of the tested implementation code.